system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -535,8 +535,7 @@ void CLdapSecManager::init(const char *serviceName, IPropertyTree* cfg)
     m_pp.setown(pp);
     int cachetimeout = cfg->getPropInt("@cacheTimeout", 5);
 
-    bool sharedCache = cfg->getPropInt("@sharedCache", true);
-    if (sharedCache)
+    if (cfg->getPropBool("@sharedCache", true))
```

Comment:
lets redbook the old behavior, and perhaps document how to create separate caches in anybody would ever want to do so